packages/sdks/react-sdk/src/types.ts

```diff
@@ -14,7 +14,7 @@ import UserProfileWidget from '@descope/user-profile-widget';
 import ApplicationsPortalWidget from '@descope/applications-portal-widget';
 import createSdk from './sdk';
 
-declare module 'react' {
+declare global {
```

Comment:
Did we check it also with React 19?
To make sure there are no type errors

Reply:
I have a react ts app that I built with vite and it works well there 👍